### DISCUSSION TITLE
Add Google Drive Integration

### DISCUSSION THREAD

**Discussion by Knowaaaa** [twentyhq/twenty#7513] [Ideas]
We use Google Workspace for our business and it would be nice if we could leverage our near infinite storage space in our share drives for saving client files. The feature I am envisioning is being able to add a column to a table which is a "Google Drive" type, which when created will create a drive folder for that specific table record in a predefined location in our google drive IE: "root/clients/Smith, Bob". 
  
Saving documents in Drive rather than directly in Twenty makes it a bit easier to work with files in other apps as we can mount Google Drive directly inside of our PDF signing app for example.

**Comment by BOHEUS** [comment]
@Knowaaaa thanks for the suggestion, soon it'll be possible to write an integration once SDK is stable and v1.0 is released (in the meantime, feel free to check out docs https://docs.twenty.com/developers/extend/apps/getting-started) and files are available as standard object and not system, I'm closing this discussion in favor of collective discussion of integrations https://github.com/twentyhq/twenty/discussions/16735
